src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/controller/PeopleController.kt

```diff
@@ -30,4 +31,18 @@ constructor(
       personService
         .getSentenceDetails(prisonNumber),
     )
+
+  override fun getCurrentUserCaseloads(allCaseloads: Boolean): ResponseEntity<List<CaseLoad>> = ResponseEntity.ok(
```

Comment:
I know Prisoners are people and users are People but... I feel like they are two very separate entities. I think we should move the user stuff into it's own test and controller etc.